3주차/1254/1254_python_제로.py

```diff
@@ -0,0 +1,22 @@
+import sys
+
+s = sys.stdin.readline()[:-1]
+
+count = 0
+def check(value):
+    global count
+    right = len(value) - 1
+    left = 0
+    while left <= right:
+        if value[left] != value[right]:
+            return False
+        left += 1
+        right -= 1
+    return True
+
+for i in range(len(s)-1):
+    checkValue = s+s[:i][::-1]
+    if check(checkValue):
+        print(len(checkValue))
+        exit(0)
+print(len(s)*2-1)
```

Comment:
```suggestion
for i in range(len(s)-1):
    checkValue = s+s[:i][::-1]
    if check(checkValue):
        print(len(checkValue))
        break
else:
    print(len(s)*2-1)
```
별건 아니구 그냥 for-else문이 있다는 걸 알려드리려고 코멘트 달아요ㅎ